src/site/content/en/blog/display-override/index.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: post
+title: Preparing for the display modes of tomorrow
+subhead: PWAs can use the "display_override" property to deal with special display modes.
+authors:
+  - thomassteiner
+date: 2021-02-16
+description: |
+  The display_override property allows developers to define a customized fallback chain of modes
+  their PWAs should be displayed in.
+hero: hero.jpg
+alt: Web App Manifest source code excerpt.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - progressive-web-apps
+---
+
+A [Web App Manifest](/add-manifest/) is a JSON file that tells the browser about your Progressive
+Web App and how it should behave when installed on the user's desktop or mobile device.
+Via the [`display`](/add-manifest/#display) property, you can customize what browser UI is shown when your app is launched. For example, you can hide the
+address bar and browser chrome. Games can even be made to launch full screen.
+As a quick recap, below are the display modes that are currently specified.
```

Comment:
```suggestion
As a quick recap, below are the display modes that are specified at the time this article was written.
```

Reply:
I think your version is too easily interpreted as 'we check this regularly and keep it up to date.'